charts/iom/README.rst

```diff
@@ -145,3 +146,22 @@ Known Defects
 |        |                                                                                                |
 +--------+------------------------------------------------------------------------------------------------+
 
+Helm parameter *oms.mailResourcesBaseUrl* was removed.
+       
+=============
+Known Defects
```

Comment:
Why do you add another `Known Defects` chapter?

Reply:
Thank you! Must have happened during rebase (resolve conflicts) and I did not recognized it. I will fix it and update the pull request tomorrow.

Reply:
Is fixed now.